views/offramp/confirmation.pug

```diff
@@ -0,0 +1,16 @@
+extends ../base
+
+block variables
+  -var title = __('You have reached the end of the prototype')
+
+block content
+
+  h1 #{title}
+ 
+  div
+    p #{__('To continue the survey click anywhere outside this popup window')}
+    p #{__('Clicking')} <i>#{__('Continue')}</i> #{__(`, will proceed to the next step, where we'll ask for feedback on your experience.`)}
+
+  a(href='/feedback' role='button' draggable='false') #{__('Continue')}
```

Comment:
We can't put a continue button on this page. they just have to click off the modal window.

Reply:
I thought the trello card said to link it to the feedback page, but I think the content was actually describing something that happens outside of the prototype